doc/changelog/1.X.X-changelog

```diff
@@ -1,3 +1,17 @@
+1.2.2 (September 05, 2019)
```

Comment:
@oesteban If you're pushing a new tag, you might want to update the date before tagging.